src/components/movie/movie.js

```diff
@@ -3,25 +3,28 @@ import LazyImage from './img';
 import MovieMain from './main';
 import MovieOverview from './overview';
 import MovieStates from './states';
-import MovieFavBtn from './favBtn';
+import MovieFavBtn from './likeBtn';
+import ShowMsg from '../showmsg';
 
 const Movie = (props) => {
-    let filteredMovies = props.movie.filter(movie => {
+    let {movie, handleFav} = props
+    console.log(props)
+    let filteredMovies = movie.filter(movie => {
         return movie.title.toLowerCase().indexOf(
         props.query.toLowerCase()) !== -1 ;
         })
-    return (
-        filteredMovies.map(movieData => (
-            <div className="row shadow mb-4" key={movieData.id}>
-                <LazyImage src={`https://image.tmdb.org/t/p/w440_and_h660_face${movieData.poster_path}`} />
+    return (filteredMovies.length > 0 ?
```

Comment:
this component must be shared between homePage movies and filgtered movies, so it will be presentation comp, why you make it to check for filteredMovies ?

Reply:
what should i do ?